challenge_take_out_the_garbage/src/challenge_take_out_the_garbage/measure_garbage.py

```diff
@@ -0,0 +1,51 @@
+# ROS
+import rospy
+import smach
+
+# TU/e
+import robot_smach_states as states
+import robot_smach_states.util.designators as ds
+import robot_smach_states.manipulation as manipulation
+from robot_skills.arms import PublicArm
+
+from geometry_msgs.msg import WrenchStamped
+import math
+
+
+class MeasureForce(object):
+    """
+    Measure the three forces in the arm
+    """
+    def __init__(self, robot):
+        """
+
+        """
+        smach.State.__init__(self, outcomes=["succeeded", "failed"])
+        self._robot = robot
+
+    def get_force(self):
+        ft_sensor_topic = '/hero/wrist_wrench/raw'
```

Comment:
please still use robot.robot_name. Although it is only possible on HERO. But keep the software dynamic so it can be used by other robots as well.